src/plugins/data-ajax/reports/pre-a11y-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/data-ajax/data-ajax-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Data Ajax",
```

```suggestion
        "dct:description": "Data ajax plugin state in basic HTML mode and when it has imported content in the page. The imported content are out of scope, only the data-ajax wrapper is.",
```
This is a description of the test subject and not of the whole pre-assessment report.